content/en/docs/setup/other_config/features/server-group-launch-settings/aws-ec2/index.md

```diff
@@ -0,0 +1,35 @@
+---
+title:  "AWS EC2 Server Group Launch Settings"
+linkTitle:  "AWS EC2 Server Group Launch Settings"
+weight: 1
```

Comment:
```suggestion
title:  "AWS EC2 Server Group Launch Settings"
linkTitle:  "AWS EC2 Server Group Launch Settings"
weight: 1
description: An AWS EC2 Server Group offers a number of launch setting configurations that can enhance your experience with Spinnaker.
```

Reply:
Is it okay for the description to be repeated under `Overview`?